threads.js.org

andywer opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

andywer commented:
Hey @MattIPv4!

I thought this would not be necessary anymore, since GitHub introduced the `custom domain` setting in the repo settings? I can change it, of course.

Thanks for getting back so quickly!

MattIPv4 commented:
@andywer The custom domain setting is just an abstraction of creating the file. In the background it still creates the file a commits it to your repo.

andywer commented:
Didn't realize! Well, I added it this morning via the setting, but reverted the change again, because it breaks the current deployment until this PR is merged.

Reverted the revert now :)
Thanks!

andywer commented:
Is there an ETA when this will get merged? The current deployment will be broken until this PR is merged. Or we revert the CNAME commit again and add it back later after this has been merged.

MattIPv4 commented:
@andywer Hiya, you will need to wait until @indus can add your request to the dns zonefile and merge this request. Unfortunately, I cannot provide an ETA for that.

indus commented:
Welcome https://threads.js.org

andywer commented:
Thanks, guys! :)
